components/layout/FullHeightComponent.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import { ReactNode, useEffect, useState } from 'react';
+
+const FullHeightContainer = ({ children }: { children: ReactNode }) => {
+  const [height, setHeight] = useState(window.innerHeight);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setHeight(window.innerHeight);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const style = {
+    height: height + 'px',
+    overflow: 'auto', // Add this style if you want the component to be scrollable
```

Comment:
```suggestion
    overflow: 'auto',
```
Don't need that comment do we?